### PR TITLE
docs(crashlytics): improve Crashlytics docs in response to #2724

### PR DESCRIPTION
**Pull request by andersonaddo**
I believe there should also be a place in the documentation that should explicitly explain that the firebase.json file is and where it should go.
Any idea where I should put that?

### CODE REVIEW

File: docs/crashlytics/quick-start.md [hunk 1]
```diff
@@ -119,4 +119,8 @@ Once your user has consented, enable Crashlytics collection via the JavaScript A
 await firebase.crashlytics().setCrashlyticsCollectionEnabled(true);
 ```
 
-To learn more about all the `firebase.json` options, view the <Anchor version group="app" href="/reference/firebasejsonconfig">documentation here</Anchor>.
+
+### Enabling Debug Reports
+By defualt, Crashlytics won't report crashes and logs from builds in debug mode. To enable it, set the `crashlytics_debug_enabled` option to true in the `firebase.json` file.
```
Comment (Salakar) [inline docs/crashlytics/quick-start.md]:
```suggestion
By default, Crashlytics won't report crashes and logs from builds in debug mode. To enable it, set the `crashlytics_debug_enabled` option to `true` in the `react-native` section of your `firebase.json` config file.
```

Comment (Salakar) [reply docs/crashlytics/quick-start.md]: Just a small suggestion.

**Comment by Salakar** [comment]
Thanks for this, small review above but after that I'm happy to merge.

**Comment by andersonaddo** [comment]
All done, @Salakar!

**Comment by Salakar** [comment]
Thanks!

**Pull request merged**